crates/api_models/src/refunds.rs

```diff
@@ -168,6 +208,71 @@ pub struct RefundResponse {
     pub charges: Option<ChargeRefunds>,
 }
 
+#[cfg(feature = "v1")]
+impl RefundResponse {
+    pub fn get_refund_id_as_string(&self) -> String {
+        self.refund_id.clone()
+    }
+}
+
+#[cfg(feature = "v2")]
+#[derive(Debug, Clone, Eq, PartialEq, Deserialize, Serialize, ToSchema)]
+pub struct RefundResponse {
+    /// Global Refund Id for the refund
+    #[schema(value_type = String)]
+    pub id: common_utils::id_type::GlobalRefundId,
+    /// The payment id against which refund is initiated
+    #[schema(value_type = String)]
+    pub payment_id: common_utils::id_type::PaymentId,
+    /// Unique Identifier for the Refund. This is to ensure idempotency for multiple partial refunds initiated against the same payment.
+    #[schema(
+        max_length = 30,
+        min_length = 30,
+        example = "ref_mbabizu24mvu3mela5njyhpit4",
+        value_type = String,
+    )]
+    pub merchant_reference_id: common_utils::id_type::RefundReferenceId,
+    /// The refund amount
+    #[schema(value_type = i64 , minimum = 100, example = 6540)]
+    pub amount: MinorUnit,
+    /// The three-letter ISO currency code
+    #[schema(value_type = Currency)]
+    pub currency: common_enums::Currency,
+    /// The status for refund
+    pub status: RefundStatus,
+    /// An arbitrary string attached to the object
+    pub reason: Option<String>,
+    /// Metadata is useful for storing additional, unstructured information on an object
+    #[schema(value_type = Option<Object>)]
+    pub metadata: Option<pii::SecretSerdeValue>,
+    /// The code for the error
+    pub error_code: Option<String>,
+    /// The error message
+    pub error_message: Option<String>,
+    /// The timestamp at which refund is created
+    #[serde(with = "common_utils::custom_serde::iso8601::option")]
+    pub created_at: Option<PrimitiveDateTime>,
```

Comment:
These fields, created_at and updated_at are not optional, these are mandatory fields in the response. We will not have any case where created_at and updated_at will be None